pom.xml

```diff
@@ -286,12 +286,21 @@
                     <group>JBoss SET/Tools</group>
                     <packager>JBoss SET</packager>
                     <mappings>
+                        <!-- not very elegant, but only way I found to ensure the proper files
+                             attributes were set was to "override" the first mapping with a second
+                             one ... -->
                         <mapping>
                             <directory>/usr/share/${project.name}</directory>
                             <filemode>644</filemode>
-                            <username>jboss</username>
-                            <groupname>jboss</groupname>
-                            <artifact />
+                            <username>root</username>
```

Comment:
See my comment on the root issue: https://github.com/jboss-set/prbz-overview/issues/20

In short, the "others" are doing it this way..